girder/utility/acl_mixin.py

```diff
@@ -113,51 +186,58 @@ def requireAccess(self, doc, user=None, level=AccessType.READ):
                 userid = str(user.get('_id', ''))
             else:
                 userid = None
-            raise AccessException("%s access denied for %s %s (user %s)." %
-                                  (perm, self.name, doc.get('_id', 'unknown'),
-                                   userid))
+            raise AccessException('%s access denied for %s %s (user %s).' %
+                                  (perm, self.name, doc.get('_id', 'unknown'), userid))
 
     def requireAccessFlags(self, doc, user=None, flags=None):
         """
         See the documentation of AccessControlledModel.requireAccessFlags, which this wraps.
```

Comment:
This no longer wraps `AccessControlledModel.requireAccessFlags`, so the documentation should be updated.